src/main/java/culturemedia/controller/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controller;
+
+import java.util.*;
+
+import culturemedia.exception.VideoNotFoundException;
+import culturemedia.model.Video;
+import culturemedia.service.CultureMediaServiceImpl;
```

Comment:
is import culturemedia.service.CultureMediaService
import the interface, not the impl